libraries/botbuilder-dialogs-adaptive/src/input/oauthInput.ts

```diff
@@ -78,9 +78,9 @@ export class OAuthInput extends InputDialog {
 
     /**
      * Called when a prompt dialog is pushed onto the dialog stack and is being activated.
```

Comment:
Should we use the Dialog class reference here?, more lines with this point on this file.